socranop/data/bash-completion/socranop-ctl

```diff
@@ -0,0 +1,20 @@
+# bash completion script for the socranop-ctl command   -*- shell-script -*-
+
+_socranop_ctl()
+{
+    # $1 is the command name
+    # $2 is the word being completed
+    # $3 is the preceding word
+    case "$3" in
+	socranop-ctl | */socranop-ctl)
+	    COMPREPLY=($(compgen -W "-h --help --version -v --verbose --no-dbus -l --list -s --set" -- "$2"))
```

Comment:
Here I have concerns about more future drift as we add/remove options in the future.

Is there a way we could have each tool generate its own compgen output at installtime so it's guaranteed to be accurate?   A good example of this is the kubectl tool which lets you run `kubectl completion bash` and the output is the bash completion logic exactly tailored to that specific version of kubectl.

(Maybe not required for 0.5, but should be done soon, else we'll hate ourselves later!)

Reply:
Make this an issue for the v0.6 milestone?